ivadomed/loader/utils.py

```diff
@@ -722,6 +726,69 @@ def reorient_image(arr, slice_axis, nib_ref, nib_ref_canonical):
     return nib.orientations.apply_orientation(arr_ras, trans_orient)
 
 
+def merge_bids_datasets(BIDS_dataset_paths_list):
```

Comment:
if `BIDS_dataset_paths_list` is a variable should be all small caps (snake_case)-- pls correct everywhere

Reply:
Fixed here: https://github.com/ivadomed/ivadomed/pull/588/commits/b35ade046cc9e65e574f45f197867d5d73f77240